engine/websocketroutine_manager.go

```diff
@@ -131,34 +137,32 @@ func (m *websocketRoutineManager) websocketRoutine() {
 
 // WebsocketDataReceiver handles websocket data coming from a websocket feed
 // associated with an exchange
-func (m *websocketRoutineManager) WebsocketDataReceiver(ws *stream.Websocket) {
-	if m == nil || atomic.LoadInt32(&m.started) == 0 {
-		return
-	}
```

Comment:
Why was this section removed?

Reply:
I just moved them to the calling function [here](https://github.com/thrasher-corp/gocryptotrader/pull/935/files/ede080485037d2fa41bd4b8e6a6edf2de3b77042#diff-ced91e5af7a15f7c959c32b7b5a2ffb26bd9771a25799af41c45acc02de1d3e5R118). The nil check is already done there in that scope and I unexported the `webscoketDataReceiver` itself.

Reply:
I think it would be best to keep it, if it's called outside of the normal path, we have a function that is stuck